docs/building.md

```diff
@@ -26,27 +27,86 @@ The entire suite of tests can be run using the following command:
 bazel test //src/test/...
 ```
 
-### MacOS
```

Comment:
Are we removing the MacOS section in favor of a more general "Containerized Builds" section?

Reply:
Yes, as there's nothing MacOS-specific anymore. In fact, the more complicated setup is for hybrid development on a Linux host.

For developing on a MacOS host, the least complex solution may be installing and running IntelliJ within the container, e.g. using https://gist.github.com/paul-krohn/e45f96181b1cf5e536325d1bdee6c949.

As an aside, we use Reviewable for all of our code reviews. Please leave any comments in there rather than using GitHub's UI.